src/SprykerEco/Glue/UnzerRestApi/Plugin/GlueApplication/UnzerNotificationResourceRoutePlugin.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi\Plugin\GlueApplication;
+
+use Generated\Shared\Transfer\RestUnzerNotificationsAttributesTransfer;
+use Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceRouteCollectionInterface;
+use Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceRoutePluginInterface;
+use Spryker\Glue\Kernel\AbstractPlugin;
+use SprykerEco\Glue\UnzerRestApi\UnzerRestApiConfig;
+
+class UnzerNotificationResourceRoutePlugin extends AbstractPlugin implements ResourceRoutePluginInterface
+{
+    /**
+     * @var string
+     */
+    protected const CONTROLLER_NAME = 'unzer-notification';
```

Comment:
```suggestion
    protected const CONTROLLER_NAME = 'unzer-notifications';
```

this code is currently not workable
please check it before passing to review

Reply:
Changes done.